.github/workflows/check.yml

```diff
@@ -0,0 +1,41 @@
+# Based upon:
+#   https://github.com/actions/starter-workflows/blob/main/ci/go.yml
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        go-version: [1.17, 1.18]
```

Comment:
###         go-version: [1.18, 1.19]

Reply:
Thanks, missed the new release.

Reply:
thinking about it, probably best to support only 1.19. No point in supporting old releases.

Reply:
I don't think that's true. Customers will use this library and we can't dictate to them that they upgrade. So we have to know that our code works with the versions of go that they might be using. See for example https://github.com/ably/ably-go#supported-versions-of-go

Reply:
fair enough. So we should be supporting 1.18 and 1.19 and should drop support of 1.17.